.github/workflows/scripts/slack_notifications/gem_notifier.rb

```diff
@@ -59,13 +59,24 @@ def self.gem_source_code_uri(gem_name)
     info = HTTParty.get("https://rubygems.org/api/v1/gems/#{gem_name}.json")
     raise "Response unsuccessful: #{info}" unless info.success?
 
-    info["source_code_uri"]
+    source_code_uri = info["source_code_uri"]
+    validate_github_url(source_code_uri)
+    source_code_uri
```

Comment:
Adding a blank line here would be great to match our internal style. 
```suggestion

    source_code_uri
```